WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -97,22 +100,23 @@ private struct ProductsSection: View {
                     .headlineStyle()
 
                 // TODO: Add a product row for each product added to the order
-                let viewModel = ProductRowViewModel(id: 1,
+                let productRowViewModel = ProductRowViewModel(id: 1,
                                                     name: "Love Ficus",
                                                     sku: "123456",
                                                     price: "20",
                                                     stockStatusKey: "instock",
                                                     stockQuantity: 7,
                                                     manageStock: true,
                                                     canChangeQuantity: true) // Temporary view model with fake data
-                ProductRow(viewModel: viewModel)
+                ProductRow(viewModel: productRowViewModel)
 
                 Button(NewOrder.Localization.addProduct) {
                     showAddProduct.toggle()
                 }
                 .buttonStyle(PlusButtonStyle())
                 .sheet(isPresented: $showAddProduct) {
-                    AddProduct(isPresented: $showAddProduct)
+                    let addProductViewModel = AddProductViewModel(siteID: viewModel.siteID)
+                    AddProduct(isPresented: $showAddProduct, viewModel: addProductViewModel)
```

Comment:
I wonder if this view should be renamed to `AddProductToOrder` or it is built to be reused across several modules?

Reply:
That's a good idea 👍  In theory it could be reused but in practice I've seen that we have different requirements for the product list in different parts of the app, so for now at least it's specific to adding products to an order.

Reply:
Renamed the view and view model in 314c4ee.